src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1235,6 +1235,25 @@ var TorrentsTable = new Class({
             }
         }
 
+        const tagHashInt = parseInt(tagHash);
+        if (!isNaN(tagHashInt)) {
```

Comment:
The double negative confuses my brain. Can this be stored in a variable for clarity? Like `const isNumber`?